### ISSUE TITLE
Add PDF to technologies

### ISSUE THREAD

**Issue by julezrulez** [w3c/wai-wcag-em-report-tool#53]
In the current list in Step 2 you could check several techniques. I always wondered why PDF is not in that list. Could PDF added to the list of check boxes?

I know you could do that by hand, but in the years that I do audits I have never added an technique by hand. Checkboxes are much easier.

**Comment by hidde** [comment]
@nitedog I agree with @julezrulez… feels like it makes sense to add PDF to our default list of technologies, what do you think?

**Comment by hidde** [comment]
Have added PDF tentatively.

**Comment by shawna-slh** [comment]
Please leave this open for other input.

**Comment by hidde** [comment]
From the [WCAG definition of “technologies”](https://www.w3.org/TR/WCAG21/#dfn-technologies):

 > Some common examples of Web content technologies include HTML, CSS, SVG, PNG, PDF, Flash, and JavaScript. 

As PDF is in that list, it seemed to me like a non-trivial change.

**Comment by bakkenb** [comment]
I agree to the addition of PDF to the list as part of the web technologies. I think this is important because PDFs, especially automated PDFs like reports and confirmations, are easily overlooked as "part of the site/application."
